### DISCUSSION TITLE
How to configure the "Resource List" when running Dashboard in standalone mode

### DISCUSSION THREAD

**Discussion by hu-srp** [microsoft/aspire#7379] [Q&A] [answered]
For the independent Aspire Dashboard, how can I configure the application so that the Dashboard can display the "Resource List" and "Console Log"? I set the environment variable "DOTNET_RESOURCE_SERVICE_ENDPOINT_URL" when running the Docker container of Dashboard. Although Dashboard has a "resource list", the list has no application

**Comment by davidfowl** [comment, marked as answer]
See https://github.com/dotnet/aspire/discussions/6772
